### PR TITLE
Genre Is Optional

### PR DESCRIPTION
**Pull request by lbedner**
Some deliveries have tracks that don't have `<Genre>`, but we shouldn't block these products from making their way through the pipeline, so here, we're making it `Optional`.

### CODE REVIEW

**Comment by ajbeach2** [comment]
Should we add a test for this?

**Comment by lbedner** [comment]
> Should we add a test for this?

Yeah, that makes sense, good call.

**Comment by lbedner** [comment]
@ajbeach2 I modified an existing test that checks for `Optional` fields.

File: tests/data/schema/valid-nulled-schema.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
             },
             "duration": "PT3M55S",
             "explicitLyrics": false,
-            "genre": "Deutschspr. HipHop /- Rap",
```
Comment (ajbeach2) [inline tests/data/schema/valid-nulled-schema.json]: Should we add another schema that includes the genre? so we have a test for when it's null and when it's not null?

**Comment by e-agbanyo** [comment]
My only comment is if this is a 2.2.0 -> 2.3.0 change of a 2.2.1 change?

File: tests/data/schema/valid-nulled-schema.json (hunk 1 above)
Comment (lbedner) [reply tests/data/schema/valid-nulled-schema.json]: I only nulled out one `genre` here. There is another one that has a value.

**Comment by lbedner** [comment]
> My only comment is if this is a 2.2.0 -> 2.3.0 change of a 2.2.1 change?

@e-agbanyo I based it off of the last revision, where we made `copyright` `Optional`, and we bumped up to `2.2.0`.

https://github.com/iheartradio/pipeline/blob/70021fe7db23367ad67af497681a2a2a0b3c8648/docs/changes.rst#version-220

File: tests/data/schema/valid-nulled-schema.json (hunk 1 above)
Comment (ajbeach2) [reply tests/data/schema/valid-nulled-schema.json]: 👍
